src/frontend/web_application/src/components/ContactProfile/components/ContactProfileForm/index.jsx

```diff
@@ -73,6 +73,10 @@ class ContactProfileForm extends Component {
 
   render() {
     const { __ } = this.props;
+    const dateProps = {};
+    if (this.state.contact.infos.birthday) {
+      dateProps.selected = new Date(this.state.contact.infos.birthday);
```

Comment:
mm yes!
here, when mounted, `this.state.contact.infos.birthday` format is like `2016-05-09T15:01:42.116000` and become a Date instance only on birthday change. I guess this is not a good way to proceed. I'll try to improve.